dont-worry-be-happy/dwbh-api
# Use email for adding user to group (and refactor)

![](https://media3.giphy.com/media/d31wuLJDCbQYersY/giphy.gif?cid=3640f6095c9a12ab7a43423073bf793d)

## HOW TO TEST
1. Add an user to a group using the email
2. Check for failures (user don't exist, group don't exists, current user is not an admin, user already belongs to group)
